ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -1073,15 +1074,15 @@ private class PostgresLedgerDao(
                     "package" -> p._1.toByteArray
                 )
               )
-            executeBatchSql(
-              SQL_INSERT_PACKAGE,
-              namedPackageParams
-            )
-            PersistenceResponse.Ok
-          }.recover {
-            case NonFatal(e) if e.getMessage.contains("duplicate key") =>
-              conn.rollback()
+            import scala.math.min
+            val changeCount = executeBatchSql(SQL_INSERT_PACKAGE, namedPackageParams)
+              .map(min(0, _))
```

Comment:
I believe the failing test in CI is because of this. You can run
```bash
bazel test //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_PostgresDaoSpec.scala
```
to test locally.